packages/utils/README.md

```diff
@@ -41,13 +41,28 @@ and exception.
 ### dom
 
 * `createHTMLDocument`: Create an HTMLDocument object from an string.
+* `determineMediaTypeBasedOnFileExtension`: Get a mime-type associated
```

Comment:
Shouldn't these be under a `### contentType` section?

Reply:
Yeah... bad rebase. Nice catch!